packages/flame_tiled/lib/src/renderable_layers/renderable_layer.dart

```diff
@@ -106,11 +106,10 @@ abstract class RenderableLayer<T extends Layer> {
   /// position.
   /// https://doc.mapeditor.org/en/latest/manual/layers/#parallax-scrolling-factor
   void applyParallaxOffset(Canvas canvas, CameraComponent camera) {
-    final anchor = camera.viewfinder.anchor;
     final cameraX = camera.viewfinder.position.x;
     final cameraY = camera.viewfinder.position.y;
-    final viewportCenterX = camera.viewport.size.x * anchor.x;
-    final viewportCenterY = camera.viewport.size.y * anchor.y;
+    final viewportCenterX = camera.viewport.size.x / 2;
+    final viewportCenterY = camera.viewport.size.y / 2;
```

Comment:
Did you test this in Flame and compare it to Tiled? I originally verified everything lined up by hand